docs/source/topics/assembly_troubleshooting.adoc

```diff
@@ -3,10 +3,9 @@
 
 [NOTE]
 ====
-The goal of {rh-prod} is to deliver an OpenShift environment for development and testing purposes.
-Issues occurring during installation or usage of specific OpenShift applications are outside of the scope of {prod}.
+The goal of {rh-prod} is to deliver an {ocp} environment for development and testing purposes.
+Issues occurring during installation or usage of specific {openshift} applications are outside of the scope of {prod}.
 Report such issues to the relevant project.
-For example, OpenShift tracks issues on link:https://github.com/openshift/origin/issues[GitHub].
```

Comment:
Removal of this line is intentional?

Reply:
It is, mostly due to the fact that I honestly don't know where we should direct users for issues upstream. The repository we originally linked to seems to have a completely different focus now. Should we link to OKD for upstream issue reporting?